docs/conf/portland/2020/sponsors/online-prospectus.rst

```diff
@@ -0,0 +1,169 @@
+:template: {{year}}/generic.html
+
+Online Sponsorship Prospectus
+#############################
+
+.. contents:: Sections
+   :local:
+   :depth: 1
+   :backlinks: none
+
+Introduction
+============
+
+Welcome to our brand new **Online Conference Sponsorship Prospectus**.
+This document has been adapted from the original :doc:`prospectus` for our Portland 2020 conference to reflect the virtual conference,
+including new benefits and increased value across our community.
+
+We're excited to work with the organizations in our community to build the best possible event in 2020.
+In particular, we would love your feedback on sponsorship levels and benefits.
+We expanded the offerings and introduced a lot of new things,
+but please let us know if there are other points of interaction with our community that would be valuable for you.
+
+Concept
+=======
+
+{% include "conf/sponsorship/concept.rst" %}
+
+Demographics
+============
+
+{% include "conf/sponsorship/demographics.rst" %}
+
+Why Sponsor
+===========
+
+{% include "conf/sponsorship/why.rst" %}
+
+Sponsorship Packages
+====================
+
+The following options are suggested sponsorship levels. We are happy to discuss adjustments and custom packages.
+
+.. warning:: This is a draft prospectus and is subject to change.
```

Comment:
This warning doesn't actually render nicely.

![Screenshot 2020-05-10 at 18 15 09](https://user-images.githubusercontent.com/98594/81504544-440e1e80-92ea-11ea-84a0-460b053334bb.png)

Reply:
Yea, the annoyances of not using a proper Sphinx theme.

Reply:
We can remove them since I'm about to merge 👍